### PR TITLE
Adding Open IE and fixing dependency parsing

### LINKED ISSUE #74: Issues with dependency parsing in models.html
I think that there are a some issues with how dependency parsing appears in models.html:

* It appears below the user gallery and how to contribute (instead of with the other models).
* There's no link to it from the sidebar.
* It doesn't have a `View Demo` link.

### PR DESCRIPTION
**Pull request by gabrielStanovsky**
- Add Open IE to the models page.
- Fix #74

### CODE REVIEW

File: models.html [hunk 1]
```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
...
+                        --predictor=open-information-extraction</code></pre></div>
+                </div>
+                <div data-tab="evaluation" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The Open Information extractor was evaluated on the <a href="https://github.com/gabrielStanovsky/oie-benchmark" target="_blank">OIE2016 corpus</a>.</p>
+                  </div>
+                </div>
+                <div data-tab="training" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The Open Information extractor was evaluated on the <a href="https://github.com/gabrielStanovsky/oie-benchmark" target="_blank">OIE2016 corpus</a>.</p>
```
Comment (DeNeutoy) [inline models.html]: The data is freely available right? You should upload it to the allennlp s3 bucket and add a config https://github.com/allenai/allennlp/tree/master/training_config here which replicates the training proceedure for this model, if possible.

File: models.html [hunk 2]
```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
...
+        </div>
+      </div>
+
+
+      <!-- Open Information Extraction-->
+      <div class="banner banner--hr">
+        <div class="constrained constrained--med">
+          <div class="padded-med">
+            <h2 id="open-information-extraction">Open Information Extraction</h2>
+            <p class="t-sm">
```
Comment (DeNeutoy) [inline models.html]: Can you add some performance statistics here?

File: models.html [hunk 3]
```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
               </div>
             </div>
 
+      <!-- Dependency Parsing-->
```
Comment (DeNeutoy) [inline models.html]: Ah nice one, thanks.

File: models.html (hunk 2 above)
Comment (gabrielStanovsky) [reply models.html]: Done.

**Comment by gabrielStanovsky** [comment]
Thanks @DeNeutoy, I addressed your comments.
Regarding data availability - unfortunately it's not publicly available and requires LDC license (PTB texts).
I added a more explicit note on that.

**Pull request merged**